<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFRDRImagingFCSMaskBuilder</class>
 <widget class="QDialog" name="QFRDRImagingFCSMaskBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Selection/Mask builder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>only those pixels will be selected, where all the conditions below apply</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,3,0,1,1">
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="4">
      <widget class="QFDoubleEdit" name="spinT32">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="cmbMode1">
       <item>
        <property name="text">
         <string>in-range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>out-of-range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;= threshold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;= threshold</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QFDoubleEdit" name="spinT12"/>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>range start/threshold</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>mode</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="cmbMode3">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>in-range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>out-of-range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;threshold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;threshold</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="cmbMode2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>in-range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>out-of-range</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;threshold</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;threshold</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QFDoubleEdit" name="spinT11"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cmbP3">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>range end</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbP2">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbP1"/>
     </item>
     <item row="2" column="4">
      <widget class="QFDoubleEdit" name="spinT22">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QFDoubleEdit" name="spinT31">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>parameter</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QFDoubleEdit" name="spinT21">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="chkMask">
       <property name="text">
        <string>non-masked only</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chkUse2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chkUse3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QFDoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>qfdoubleedit.h</header>
   <slots>
    <signal>valueChanged(double)</signal>
    <slot>setValue(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbP1</tabstop>
  <tabstop>cmbMode1</tabstop>
  <tabstop>spinT11</tabstop>
  <tabstop>spinT12</tabstop>
  <tabstop>chkUse2</tabstop>
  <tabstop>cmbP2</tabstop>
  <tabstop>cmbMode2</tabstop>
  <tabstop>spinT21</tabstop>
  <tabstop>spinT22</tabstop>
  <tabstop>chkUse3</tabstop>
  <tabstop>cmbP3</tabstop>
  <tabstop>cmbMode3</tabstop>
  <tabstop>spinT31</tabstop>
  <tabstop>spinT32</tabstop>
  <tabstop>chkMask</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QFRDRImagingFCSMaskBuilder</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QFRDRImagingFCSMaskBuilder</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse2</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbP2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>22</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse2</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbMode2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse2</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinT21</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse2</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinT22</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse3</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinT32</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse3</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinT31</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>13</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse3</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbMode3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>19</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkUse3</sender>
   <signal>toggled(bool)</signal>
   <receiver>cmbP3</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>16</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
